Description: The upstream buildsystem touches files under debian/, forbid him to do so.
Author: Didier Raboud <odyx@debian.org>
Last-Update: 2011-10-21
--- a/Makefile.am
+++ b/Makefile.am
@@ -24,10 +24,6 @@
 
 ACLOCAL_AMFLAGS = -I m4
 
-noinst_DATA = \
-	debian/control \
-	debian/rules
-
 debian/control: $(srcdir)/Makefile $(srcdir)/debian/control.in
 	rm -f $@ $@.tmp; \
 	sed -e 's|[@]PACKAGE_BUGREPORT[@]|$(PACKAGE_BUGREPORT)|' \
@@ -66,14 +62,6 @@
 	README \
 	README.ja \
 	bootstrap \
-	debian/changelog \
-	debian/compat \
-	debian/control.in \
-	debian/copyright \
-	debian/docs \
-	debian/postinst \
-	debian/postrm \
-	debian/rules.in \
 	epson-inkjet-printer-escpr.spec \
 	epson-inkjet-printer-escpr.spec.in \
 	lsb/lsb-rpm.spec \
